diff --git a/config/project_config_embeddable.cmake b/config/project_config_embeddable.cmake
index 794e0cb9..18a9ab94 100644
--- a/config/project_config_embeddable.cmake
+++ b/config/project_config_embeddable.cmake
@@ -1510,9 +1510,9 @@ macro (init_build)
   if (NOT CMAKE_CXX_FLAGS)
 	#set (CMAKE_CXX_FLAGS "-Wall -Wextra -pedantic -Werror")
     if (${RUN_GCOV} STREQUAL "ON")
-	  set (CMAKE_CXX_FLAGS "-Wall -Werror -fprofile-arcs -ftest-coverage")
+	  set (CMAKE_CXX_FLAGS "-Wall -fprofile-arcs -ftest-coverage")
     else (${RUN_GCOV} STREQUAL "ON")
-      set (CMAKE_CXX_FLAGS "-Wall -Werror")
+      set (CMAKE_CXX_FLAGS "-Wall")
 	endif (${RUN_GCOV} STREQUAL "ON")
   endif (NOT CMAKE_CXX_FLAGS)
   # Tell the source code the version of Boost (only once)
